supabase/__init__.py

```diff
@@ -1,35 +1,66 @@
+from gotrue.errors import (
+    AuthApiError,
+    AuthError,
+    AuthImplicitGrantRedirectError,
+    AuthInvalidCredentialsError,
+    AuthRetryableError,
+    AuthSessionMissingError,
+    AuthUnknownError,
+    AuthWeakPasswordError,
+)
 from postgrest import APIError as PostgrestAPIError
 from postgrest import APIResponse as PostgrestAPIResponse
+from realtime import AuthorizationError, NotConnectedError
 from storage3.utils import StorageException
+from supafunc.errors import FunctionsError, FunctionsHttpError, FunctionsRelayError
```

Comment:
Not urgent: I think there was an ask for a move from `supafunc` to `supabase_functions` can do that in the future

Reply:
Yes this has to wait until 3.0 of the library as it's a breaking change also this current addition is to help with migrating to that as user's will now import the errors from the `supabase` library rather than `supafunc` as some are probably doing now.